.circleci/config.yml

```diff
@@ -126,6 +126,26 @@ references:
           owner=${CIRCLE_PROJECT_USERNAME} repo=${CIRCLE_PROJECT_REPONAME} \
           tag=${CIRCLE_TAG} ASSETS=${PACKAGES_DIR:?}/*
 
+  fail_notification: &fail_notification
+    run:
+      name: Send failure notification
+      environment:
+        HOOK_TEMPLATE: |
+          {
+            "text": "CircleCI job **%s** failed on branch **%s**",
```

Comment:
I suggest to name who did change (`CIRCLE_USERNAME`? did not try it) so to jump-start initial investigation.

Reply:
@lucafavatella The notification contains link to the CI job where you can find all the information needed.

I prefer to keep the notification params simple to easier maintenance and better stability.